axlearn/cloud/gcp/project_setup.sh

```diff
@@ -0,0 +1,71 @@
+# This is a script to set up a brand new GCP project before you use AXLearn tools"""
```

Comment:
```suggestion
# Sets up a brand new GCP project for AXLearn. See also the "Getting Started" docs linked in the main readme.
```
Should we move it under https://github.com/apple/axlearn/tree/main/axlearn/cloud/gcp/scripts?

Reply:
Yes... moved it in new commit. (But I forgot to commit suggestion first, so you might have to do it again...)